### PR TITLE
Disable export button if exheader is required but not set

### PR DESCRIPTION
**Pull request by ThanatosGit**

### CODE REVIEW

**Review by Miepee — APPROVED** [review]

**Review by henriquegemignani — APPROVED** [review]

**Comment by henriquegemignani** [comment]
actual test failures, not flaky

**Comment by ThanatosGit** [comment]
> actual test failures, not flaky

OS dependent failures...hooray

File: test/games/samus_returns/gui/dialog/test_msr_game_export_dialog.py [hunk 1]
```diff
@@ -344,6 +344,12 @@ def test_get_game_export_params_custom(skip_qtbot, tmp_path):
 
 def test_export_button_without_remote(skip_qtbot, tmp_path, mocker):
     # Setup
+    mocker.patch("platform.system", return_value="Windows")
```
Comment (Miepee) [inline test/games/samus_returns/gui/dialog/test_msr_game_export_dialog.py:347]: This stuff is only necessary until #6770 is fixed, right?

Comment (ThanatosGit) [reply test/games/samus_returns/gui/dialog/test_msr_game_export_dialog.py:347]: Yes

**Pull request merged**